pkgs/development/libraries/science/astronomy/indilib/indi-firmware.nix

```diff
@@ -45,9 +48,33 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-    for f in libfishcamp/CMakeLists.txt libsbig/CMakeLists.txt
+    for f in \
+     indi-qsi/CMakeLists.txt \
+     indi-dsi/CMakeLists.txt \
+     indi-armadillo-platypus/CMakeLists.txt \
+     indi-orion-ssg3/CMakeLists.txt \
+     libqhy/CMakeLists.txt \
+     libfishcamp/CMakeLists.txt \
+     libsbig/CMakeLists.txt \
+     */*.rules;
     do
-      substituteInPlace $f --replace "/lib/firmware" "lib/firmware"
+      substituteInPlace $f \
+        --replace "/lib/udev/rules.d" "lib/udev/rules.d" \
+        --replace "/etc/udev/rules.d" "lib/udev/rules.d" \
+        --replace "/lib/firmware" "lib/firmware" \
+        --replace "/sbin/fxload" "${fxload}/bin/fxload" \
+        --replace "/bin/sleep" "${coreutils}/bin/sleep" \
+        --replace "/bin/sh" "${bash}/bin/sh" \
+        --replace "/bin/echo" "echo" \
+        --replace "lib/firmware/qhy" "lib/firmware"
+    done
+  '';
+
+  postInstall = ''
+    for f in $out/lib/udev/rules.d/*.rules;
+    do
+      substituteInPlace $f \
+        --replace "lib/firmware" "$out/lib/firmware"
     done
```

Comment:
Is there a reason this can't be done in `postPatch`? Seems like it can be without issue, this is pretty roundabout.

Reply:
Is `$out` defined at that point?

Reply:
Yes.